docs/api_docs/bundle.yaml

```diff
@@ -1311,6 +1311,14 @@ definitions:
         x-omitempty: true
         items:
           type: string
+      flagTagsOperator:
```

Comment:
btw, this doc is generated, you can move the change to https://github.com/checkr/flagr/blob/master/swagger/index.yaml#L518, and then run `make gen` to get the same result

Reply:
Sure, I wasn't sure how this work